feature/policy_forwarding/decapsulation/otg_tests/ip_guev1_static_decap_subnet_range/ip_guev1_static_decap_subnet_range_test.go

```diff
@@ -0,0 +1,642 @@
+
+	if trafficShouldPass {
+		t.Logf("traffic validation for flow %s. Expecting Traffic TX = RX.", flowName)
+		if framesTxV4 == 0 {
+			t.Error("No traffic was generated and frames transmitted were 0")
+		} else if framesRxV4 == framesTxV4 {
+			t.Logf("Traffic validation successful for [%s] FramesTx: %d FramesRx: %d", flowName, framesTxV4, framesRxV4)
+		} else {
+			t.Errorf("Traffic validation failed for [%s] FramesTx: %d FramesRx: %d", flowName, framesTxV4, framesRxV4)
+		}
+	} else {
+		t.Logf("traffic validation for flow %s. Expecting Traffic Loss", flowName)
+		if framesTxV4 == 0 {
+			t.Error("No traffic was generated and frames transmitted were 0")
+		} else if framesRxV4 == 0 {
+			t.Logf("PASS: Traffic Validation is successful as no packets received at the destination as Expected")
+		} else {
+			t.Error("FAIL: Traffic Validation is failed as no packets expected at the destination ")
+		}
+	}
+}
+
+// startCapture starts the capture on the otg ports
+func startCapture(t *testing.T, ate *ondatra.ATEDevice) {
+	otg := ate.OTG()
+	cs := gosnappi.NewControlState()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.START)
+	otg.SetControlState(t, cs)
+}
+
+// stopCapture starts the capture on the otg ports
+func stopCapture(t *testing.T, ate *ondatra.ATEDevice) {
+	otg := ate.OTG()
+	cs := gosnappi.NewControlState()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.STOP)
+	otg.SetControlState(t, cs)
+}
+
+func enableCapture(t *testing.T, config gosnappi.Config, port string) {
+	config.Captures().Clear()
+	t.Log("Enabling capture on ", port)
+	config.Captures().Add().SetName(port).SetPortNames([]string{port}).SetFormat(gosnappi.CaptureFormat.PCAP)
+}
+
+func processCapture(t *testing.T, ate *ondatra.ATEDevice, port string) string {
+	otg := ate.OTG()
+	bytes := otg.GetCapture(t, gosnappi.NewCaptureRequest().SetPortName(port))
+	time.Sleep(captureWait * time.Second)
+	pcapFile, err := os.CreateTemp("", "pcap")
+	if err != nil {
+		t.Errorf("ERROR: Could not create temporary pcap file: %v\n", err)
+	}
+	if _, err := pcapFile.Write(bytes); err != nil {
+		t.Errorf("ERROR: Could not write bytes to pcap file: %v\n", err)
+	}
+	defer pcapFile.Close()
+	return pcapFile.Name()
+}
+
+func verfiy_policer_matched_packets(t *testing.T, dut *ondatra.DUTDevice) uint64 {
+	matchpackets := uint64(0)
+	if !deviations.PolicyRuleCountersOCUnsupported(dut) {
+		const timeout = 10 * time.Second
+		isPresent := func(val *ygnmi.Value[uint64]) bool { return val.IsPresent() }
+		_, ok := gnmi.Watch(t, dut, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).PolicyForwarding().Policy(policyName).Rule(policyId).MatchedPkts().State(), timeout, isPresent).Await(t)
+		if !ok {
+			t.Errorf("Unable to find matched packets")
+		}
+		matchpackets = gnmi.Get(t, dut, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).PolicyForwarding().Policy(policyName).Rule(policyId).MatchedPkts().State())
+	}
+	return matchpackets
```

Comment:
There needs to be an else condition with a switch dut.Vendor() that specifies some vendor specific yang path to fetch these counters.  (CLI is not permitted to be used for getting counters per [deviations style guide](https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/README.md))